help/guardrails-for-ssm-tagging/sdfgh.md

```diff
@@ -0,0 +1,600 @@
+---
+layout: help-security
```

Comment:
This file looks good, but the name is crappy name does not make sense, never practice without a meaningful target. So your help/guardrails-for-ssm-tagging/index.md should follow similar standard.